src/scala/com/github/johnynek/bazel_deps/MakeDeps.scala

```diff
@@ -78,9 +80,9 @@ object MakeDeps {
         // build the workspace
         val ws = Writer.workspace(g.depsFile, normalized, duplicates, shas, model)
         if (g.checkOnly) {
-          executeCheckOnly(model, projectRoot, IO.path(workspacePath), ws, targets, formatter)
+          executeCheckOnly(model, projectRoot, IO.path(workspacePath),targetFilePathOpt.map(e => IO.path(e)),  enable3rdPartyInRepo, ws, targets, formatter)
```

Comment:
nit: space after comma before targetFilePathOpt, and too many spaces on the next comma